internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -185,7 +225,11 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				},
 			},
 			"mongo_db_employee_access_grant": schema.SingleNestedAttribute{ // TODO: not in current resource, already in mongodbemployeeaccessgrant, will probably delete, was generated as mongo_dbemployee_access_grant
```

Comment:
i keep it just in case, but I think at this moment we can already get rid of mongo_db_employee_access_grant

Reply:
removed in 91ad6a07285ab7000f8f27d4c6ef00681b338cb9